commands/cli.go

```diff
@@ -135,27 +152,26 @@ func awsNuke(c *cli.Context) error {
 		return fmt.Errorf("You can not specify both --resource-type and --exclude-resource-type.")
 	}
 
-	// Var check to make sure only allowed resource types are included in the --resource-type or --exclude-resource-type
-	// args.
-	invalidresourceTypes := []string{}
-	for _, resourceType := range resourceTypes {
-		if resourceType == "all" {
-			continue
-		}
-		if !aws.IsValidResourceType(resourceType, allResourceTypes) {
-			invalidresourceTypes = append(invalidresourceTypes, resourceType)
-		}
-	}
+	// Check command line resource type values
+	invalidResourceTypes := map[string][]string{}
```

Comment:
Why store these in a map?

Reply:
Had kept them in a map in case we want to aggregate errors for list, exclude and ignore error types if user specifies invalid values. But there isn't a need of doing so at the moment - fixed it.